gradient/cli/deployments.py

```diff
@@ -461,3 +481,59 @@ def get_deployment(deployment_id, api_key, options_file):
     deployment_client = get_deployment_client(api_key)
     command = deployments_commands.GetDeploymentDetails(deployment_client=deployment_client)
     command.execute(deployment_id)
+
+
+@deployments_tags.command("add", help="Add tags to deployment")
+@click.argument("id", cls=common.GradientArgument)
+@click.option(
+    "--tag",
+    "tags",
+    type=str,
```

Comment:
I think str is the default value, to it doesn't have to be passed here (and in a few other places)